gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -562,7 +593,41 @@ export class AdvancedDynamicTexture extends DynamicTexture {
         this.onEndLayoutObservable.clear();
         super.dispose();
     }
+
+    /** Creates the GUI editor window. */
+    private _createGUIEditor() {
+        this.BJSGUIEDITOR.GUIEditor.Show({
+            liveGuiTexture: this
+        });
+    }
+
+    /**
+     * Launch the node material editor
+     * @param config Define the configuration of the editor
+     * @return a promise fulfilled when the node editor is visible
+     */
+     public edit(config?: IGUIEditorOptions): Promise<void> {
+        return new Promise((resolve, reject) => {
+            this.BJSGUIEDITOR = this.BJSGUIEDITOR || this._getGlobalGUIEditor();
+            if (typeof this.BJSGUIEDITOR == 'undefined') {
+                const editorUrl = config && config.editorURL ? config.editorURL : AdvancedDynamicTexture.EditorURL;
```

Comment:
Nit: config?.editorURL || AdvancedDynamicTexture.EditorURL

Reply:
Does that work?

I would rather think of `config?.editorURL ?? AdvancedDynamicTexture.EditorURL`

Reply:
it should and it would also bypass empty strings which are faulty where the "nullish" coallescing will only ever check null and undefined IIRC